projects.go

```diff
@@ -83,6 +83,7 @@ type Project struct {
 	StarCount                                 int                        `json:"star_count"`
 	RunnersToken                              string                     `json:"runners_token"`
 	PublicBuilds                              bool                       `json:"public_builds"`
+	PublicJobs                                bool                       `json:"public_jobs"`
```

Comment:
I guess we should move `PublicBuilds` to a `// Deprecated members` block as done [here](https://github.com/xanzy/go-gitlab/blob/754673b7bf0a19511b5363ccf7b60fd07ed0ebf0/runners.go#L88) for example...

Reply:
Actually, since the field isn't returned anymore by the API, what do you think of removing it completely? It's always `false` anyway, so it can lead to confusion.  
Otherwise, I'm fine moving it to a `// Deprecated members` block as you suggested.

Reply:
It's still needed if you want to talk to an older version of GitLab, so I prefer to keep it...